packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderExample.shorthand.tsx

```diff
@@ -0,0 +1,37 @@
+import { Datepicker, Flex, Text, pxToRem } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerHeaderExample = () => {
+  return (
+    <Datepicker
+      today={new Date(2020, 8, 12, 0, 0, 0, 0)}
+      calendar={{
+        header: {
+          children: (ComponentType, props) => {
+            return (
+              <Flex space={'between'} hAlign={'center'}>
+                <Datepicker.CalendarHeaderAction
+                  onClick={props.onPreviousClick}
+                  direction={'previous'}
+                  title={props.prevMonthAriaLabel}
+                  disabled={props.disabledPreviousButton}
+                  icon={{}}
```

Comment:
Is empty icon needed?

Reply:
Yep, in the current implementation of DatepickerCalendarHeaderAction we need to have an icon object, I'll think about updating it later

Reply:
Maybe we can have it as default prop then?

Reply:
Done, thank you!